cli/admin/src/test/java/io/pravega/cli/admin/controller/SecureControllerCommandsTest.java

```diff
@@ -157,5 +170,20 @@ public void testDescribeStreamCommand() {
         Assert.assertTrue(commandResult.contains("testStream"));
     }
 
+
+    @Test
+    @SneakyThrows
+    public void testDeleteReaderGroupCommand() {
+        String commandResult = TestUtils.executeCommand("controller delete-readergroup _system testRG", cliConfig());
+        Assert.assertTrue(commandResult.contains("404"));
+        Assert.assertNotNull(ControllerDescribeReaderGroupCommand.descriptor());
+
+        //execute the delete reader group command
+        commandResult = TestUtils.executeCommand("controller delete-readergroup testScope testRG", cliConfig());
```

Comment:
This test is relying on `testScope testRG` to exist, but I don't see that it is creating it. Why don't you create the reader group inside the test itself? Why do you need to create it outside this test instead of doing it in the `static` block? It makes the test not self contained.